### ISSUE TITLE
PanelMenu item focus not removed

### ISSUE THREAD

**Issue by vdbongard** [primefaces/primeng#18007]
### Describe the bug

When having multiple panel menus and clicking different items, the focus is not removed. This only happens on Firefox. On Chrome it works.

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/panelmenu#multiple

### Environment

The bug only happens in Firefox. On Chrome it works.

### Angular version

19.2.0

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Go to the PanelMenu docs page on Firefox and open the "Multiple" section (https://primeng.org/panelmenu#multiple)
2. Click on multiple different PanelMenu items in different panel menus
3. See how multiple items stay selected (Left Firefox, Right Chrome)
![Image](https://github.com/user-attachments/assets/00dfb9c4-7857-4a16-ace0-b73b83fab761)

### Expected behavior

The items should not stay focused, like it is in Chrome.

**Comment by vdbongard** [comment]
The bug still exists.
